### DISCUSSION TITLE
The given data was invalid

### DISCUSSION THREAD

**Discussion by ju5t** [statamic/cms#5083] [Q&A] [answered]
Out of the blue I'm getting errors about the data being invalid. On existing items it marked the slug as being the culprit. It said the slug already existed. I have now disabled the slug in the collection and removed it from the sidebar in my blueprint (as we don't use it), but the error is still there, just not visible.

This happens for existing items and new items.

Is there something I can do to solve this?

**Comment by jackmcdade** [comment, marked as answer]
Check your Blueprint's validation rules for the slug field – there's probably a "unique" rule in there you'll need to remove that should solve it.

**Reply by ju5t** [reply]
Thanks for the quick reply! That didn't fix it unfortunately. I re-added the slug so the error reappears and it says `This URI has already been taken.`. I thought it was mentioning it existed, sorry about that.

**Reply by ju5t** [reply]
Ah found it. Earlier today I added a route to the collection and that was wrong. I removed the route and it's fixed now!

**Reply by jackmcdade** [reply]
![woop woop](https://media1.giphy.com/media/1ThndC5odGuUU/200.gif)
